### PR TITLE
fix: xrp lang translations

### PR DESCRIPTION
**Pull request by TejasvOnly**

### CODE REVIEW

File: packages/cysync-core-constants/src/i18n/lang/en.json [hunk 1]
```diff
@@ -1040,6 +1040,16 @@
...
+      },
+      "xrp": {
+        "XRP_0000": {
+          "heading": "Some unknown error occured",
+          "subtext": "Try again and if the problem persists, contact Cypherock support for assistance"
+        },
+        "XRP_0001": {
+          "heading": "Transaction Failed",
+          "subtext": "Your transaction may have failed. Please wait a moment then check the transaction history before trying again and if the problem persists, contact Cypherock support for assistance"
+        }
```
Comment (TejasvOnly) [inline packages/cysync-core-constants/src/i18n/lang/en.json]: Do not add xrp error strings for now, we will have to add this when proper xrp server error mapping is created

File: packages/cysync-core-constants/src/i18n/types.ts [hunk 2]
```diff
@@ -590,7 +590,10 @@ export interface LangErrors {
   databaseError: IErrorMsg;
   serverErrors: Record<ServerErrorType, IErrorMsg>;
   serverCoinErrors: Partial<
-    Record<CoinFamilyWithDefault, Record<ServerCoinErrorTypes, IErrorMsg>>
+    Record<
+      CoinFamilyWithDefault,
+      Partial<Record<ServerCoinErrorTypes, IErrorMsg>>
+    >
```
Comment (TejasvOnly) [inline packages/cysync-core-constants/src/i18n/types.ts]: revert

**Review by TejasvOnly — COMMENTED** [review]
Remove all changes related to xrp errors and add a changeset in this pr, This pr should only contain the language translations for existing strings

File: packages/cysync-core-constants/src/i18n/types.ts (hunk 2 above)
Comment (muzaffarbhat07) [reply packages/cysync-core-constants/src/i18n/types.ts]: reverted

File: packages/cysync-core-constants/src/i18n/lang/en.json (hunk 1 above)
Comment (muzaffarbhat07) [reply packages/cysync-core-constants/src/i18n/lang/en.json]: removed al xrp error related strings

**Pull request merged**